src/components/app.js

```diff
@@ -223,6 +235,16 @@ const App = () => {
                                     in.
                                 </h1>
                             </Route>
+                            <Route path={'/'}>
```

Comment:
Why do you not generate one path for each process?

Reply:
Creation of a Route for each Process would not be optimal since generating a Route also creates the corresponding Process component. Creating one Route and adapting the displayed Process component to the content of the url seems to be better for display performance.

Reply:
At least you could you a variable like in gridstudy
```js
  <Route exact path="/:userId/studies/:studyName">
       <StudyPane view={STUDY_VIEWS[tabIndex]} />
  </Route>
```

Reply:
Done